nginx.tmpl

```diff
@@ -66,11 +66,12 @@ http {
         return 200;
     }
 
-    location / {
+    location ~* ^/([^/]*) {
```

Comment:
Det var et godt spørsmål egentlig. Formålet med regexen er å hente ut delen som ligger mellom første og andre slash for å bruke den i en header. (Som ifølge Bouvet som trenger den virker å fungere fint).

Men jeg har tenkt på `[^/]` som "NOT /" (basert på en stackoverflow et eller annet sted), mens det jo strengt tatt betyr noe annet.

Bedre forslag? Eller fungerer det slik?